GoogleForCreators/web-stories-wp
# Text Sets: Enable feature flag plus integration tests

## Summary

Enables Text Sets!

## Relevant Technical Choices



## To-do

None

## User-facing changes

Text Sets are now enabled by default!

## Testing Instructions

1. Edit a story
2. Switch to the text panel
3. See that text sets are present

---

#4514
